### PR TITLE
Add last updated time

### PR DESCRIPTION
**Pull request by rachx**

### CODE REVIEW

**Comment by rachx** [comment]
Fixes #131 
Should pass test cases now

File: src/main/java/seedu/agendum/storage/XmlAdaptedTask.java [hunk 1]
```diff
@@ -16,11 +16,13 @@
  */
 public class XmlAdaptedTask {
 
-    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```
Comment (burnflare) [inline src/main/java/seedu/agendum/storage/XmlAdaptedTask.java:19]: Would this render old XML files non-backward compatible?

**Review by burnflare — APPROVED** [review]
Looks great

File: src/main/java/seedu/agendum/storage/XmlAdaptedTask.java (hunk 1 above)
Comment (rachx) [reply src/main/java/seedu/agendum/storage/XmlAdaptedTask.java:19]: We used this for v0.4 too and I changed the only test case it affected
Edit: yes it is annoying when switching between branches

Comment (burnflare) [reply src/main/java/seedu/agendum/storage/XmlAdaptedTask.java:19]: Great, feel free to merge in!

**Comment by burnflare** [comment]
Cool! Feel free to merge this :)

**Pull request merged**